.github/workflows/scala.yml

```diff
@@ -132,7 +131,6 @@ jobs:
         run: |
           sleep 1
           sbt --no-colors launcher/buildNativeImage
-          echo '::set-env name=LAUNCHER_NATIVE_IMAGE_TEST_SKIP_BUILD::true'
```

Comment:
This flag should not be removed - it indicates that the Native Image is pre-built before tests and the tests do not need to rebuild it.
Automatic rebuilding is a user-facing feature (so that the developer does not forget to rebuild launcher before running tests that require it) that had to be disabled on the CI because building the native image *while* running tests uses too much memory.

(I guess I should have added a comment saying something like the above inside the job so it would not be removed by mistake)